code/modules/antagonists/eldritch_cult/knowledge.dm/flesh_lore.dm

```diff
@@ -0,0 +1,257 @@
+/datum/eldritch_knowledge/base_flesh
+	name = "Principle of Hunger"
+	desc = "Opens up the path of flesh to you. Allows you to transmute a pool of blood with a kitchen knife into a Flesh Blade"
+	gain_text = "Hundred's of us starved, but I.. I found the strength in my greed."
+	banned_knowledge = list(/datum/eldritch_knowledge/base_ash,/datum/eldritch_knowledge/base_rust,/datum/eldritch_knowledge/ash_final,/datum/eldritch_knowledge/rust_final)
+	next_knowledge = list(/datum/eldritch_knowledge/flesh_grasp)
+	required_atoms = list(/obj/item/kitchen/knife,/obj/effect/decal/cleanable/blood)
+	result_atoms = list(/obj/item/melee/sickly_blade/flesh)
+	cost = 1
+	route = "Flesh"
+
+/datum/eldritch_knowledge/flesh_ghoul
+	name = "Imperfect Ritual"
+	desc = "Allows you to resurrect the dead as voiceless dead by sacrificing them on the transmutation rune with a poppy. Voiceless dead are mute and have 50 HP. You can only have 2 at a time."
+	gain_text = "I found notes.. notes of a ritual, it was unfinished and yet i still did it."
+	cost = 1
+	required_atoms = list(/mob/living/carbon/human,/obj/item/reagent_containers/food/snacks/grown/poppy)
+	next_knowledge = list(/datum/eldritch_knowledge/flesh_mark,/datum/eldritch_knowledge/armor,/datum/eldritch_knowledge/ashen_eyes)
+	route = "Flesh"
+	var/max_amt = 2
+	var/current_amt = 0
+	var/list/ghouls = list()
+
+/datum/eldritch_knowledge/flesh_ghoul/on_finished_recipe(mob/living/user,list/atoms,loc)
+	var/mob/living/carbon/human/H
+	for(var/mob/living/carbon/human/humie in atoms)
+		H = humie
+	if(!H)
+		return
+	var/mob/dead/observer/G
+	for(var/mob/dead/observer/ghost in GLOB.dead_mob_list) //excludes new players
```

Comment:
why not let observers possess these corpses? they're ghosts too, damnit

Reply:
good catch